R/pkg/R/functions.R

```diff
@@ -963,6 +964,7 @@ setMethod("kurtosis",
 #' last(df$c, TRUE)
 #' }
 #' @note last since 1.4.0
+#' @note the function is non-deterministic because its result depends on order of rows.
```

Comment:
for the first/last maybe word it:
"the function is non-deterministic, because its results depends on order of rows, which may be non-deterministic after a shuffle"